src/config.ts

```diff
@@ -89,10 +89,19 @@ export const ASSET_SWAPPER_MARKET_ORDERS_OPTS: Partial<SwapQuoteRequestOpts> = {
     noConflicts: true,
     excludedSources: EXCLUDED_SOURCES,
     runLimit: 2 ** 15,
-    bridgeSlippage: 0.002,
+    bridgeSlippage: 0.01,
     dustFractionThreshold: 0.0025,
     numSamples: 13,
     sampleDistributionBase: 1.05,
+    fees: {
+        [ERC20BridgeSource.Uniswap]: new BigNumber(2.5e5),
+        [ERC20BridgeSource.Native]: new BigNumber(3e5),
+        [ERC20BridgeSource.CurveUsdcDai]: new BigNumber(4e5),
+        [ERC20BridgeSource.Eth2Dai]: new BigNumber(5e5),
+        [ERC20BridgeSource.CurveUsdcDaiUsdt]: new BigNumber(5e5),
+        [ERC20BridgeSource.CurveUsdcDaiUsdtTusd]: new BigNumber(8e5),
+        [ERC20BridgeSource.Kyber]: new BigNumber(8e5),
+    },
```

Comment:
nit: it should be possible to type this such that this doesn't compile unless this map is exhaustive.